### PR TITLE
Fax Relay API

### PR DESCRIPTION
**Pull request by lpradovera**

### CODE REVIEW

**Comment by lpradovera** [comment]
@bryanrite please review.

File: examples/relay/fax/fax_receive.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Signalwire::Logger.logger.level = ::Logger::DEBUG
+
+class MyConsumer < Signalwire::Relay::Consumer
+  contexts ['incoming']
+
+  def on_incoming_call(call)
+    call.answer
+    fax_result = call.fax_receive
+    logger.debug "Received a fax: #{fax_result.document} that is #{fax_result.pages} pages long"
```
Comment (bryanrite) [inline examples/relay/fax/fax_receive.rb]:
```suggestion
    logger.debug "Received a fax: #{fax_result.document} that is #{fax_result.pages} pages long."
```

**Review by bryanrite — DISMISSED** [review]
Looks great!

**Review by bryanrite — DISMISSED** [review]

File: examples/relay/fax/fax_send.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  signalwire
+].each { |f| require f }
+
+Signalwire::Logger.logger.level = ::Logger::DEBUG
+
+class OutboundConsumer < Signalwire::Relay::Consumer
+  def ready
+    call = client.calling.new_call(from: ENV['FAX_FROM_NUMBER'], to: ENV['FAX_TO_NUMBER'])
+    call.dial
+    call.fax_send("https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf")
```
Comment (bryanrite) [inline examples/relay/fax/fax_send.rb]:
```suggestion
    call.fax_send(document: "https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf")
```

**Review by bryanrite — CHANGES_REQUESTED** [review]

**Review by bryanrite — APPROVED** [review]

**Pull request merged**